filament/src/driver/vulkan/VulkanDisposer.cpp

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (C) 2019 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "driver/vulkan/VulkanDisposer.h"
+
+#include <utils/Panic.h>
+
+namespace filament {
+namespace driver {
+
+void VulkanDisposer::createDisposable(Key resource, std::function<void()> destructor) noexcept {
+    mDisposables[resource] = { 1, destructor };
```

Comment:
the initialization of the recCount to 1 should be done in the ctor -- it shouldn't be a parameter.

Reply:
Ok, made the "1" into a field initializer.  (It's a tiny POD structure so I'd rather not give it a constructor method.)